src/config/konnectors.json

```diff
@@ -1561,6 +1561,32 @@
       "bankId": "119"
     }
   },
+  {
+    "slug": "impots",
+    "name": "Impots.gouv",
+    "editor": "Cozy Cloud",
+    "vendorLink": "https://cfspart.impots.gouv.fr/",
```

Comment:
I like to put the link to the login form directly (here : https://cfspart.impots.gouv.fr/LoginMDP). This link is used by collect when the connector returns the USER_ACTION_NEEDED error. This allows the user to access the login form directly.

Reply:
OK !